content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md

```diff
@@ -74,17 +79,18 @@ service/php-apache created
 
 ## Create Horizontal Pod Autoscaler
 
-Now that the server is running, we will create the autoscaler using
-[kubectl autoscale](/docs/reference/generated/kubectl/kubectl-commands#autoscale).
-The following command will create a Horizontal Pod Autoscaler that maintains between 1 and 10 replicas of the Pods
+Now that the server is running, we will create a Horizontal Pod Autoscaler that maintains between 1 and 10 replicas of the Pods
```

Comment:
@kkanska , Thanks for the updates! Looks good.
I wonder if the [appendix-other-possible-scenarios](https://deploy-preview-18974--kubernetes-io-master-staging.netlify.com/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#appendix-other-possible-scenarios) section on this page could be removed since you replaced `kubectl autoscale` with `kubectl apply ...`?
Line 82-83, Possible edits:
With the php-apache service deployed and running, create a Horizontal Pod Autoscaler that maintains between 1 and 10 replicas of the Pods controlled by the php-apache deployment.

/lgtm

Reply:
To be honest I didn't notice this appendix, my bad. This part can as well stay as it was before my changes and probably would be more readable with the `kubectl autoscale`.